source/features/default-branch-button.tsx

```diff
@@ -40,12 +61,6 @@ async function add(branchSelector: HTMLElement): Promise<void> {
 
 	selectorWrapper.before(defaultLink);
 	groupButtons([defaultLink, selectorWrapper]).classList.add('d-flex', 'rgh-default-branch-button-group');
-
-	// Only request it later to avoid slowing down the page load
-	if (!await isUrlReachable(url.href)) {
-		defaultLink.classList.add('disabled');
-		defaultLink.setAttribute('aria-label', 'Object not found on the default branch');
-	}
```

Comment:
404s are fine, they have `useful-not-found-page` which has further actions for the user.